include/swift/Frontend/FrontendInputsAndOutputs.h

```diff
@@ -79,6 +84,9 @@ class FrontendInputsAndOutputs {
 
   bool isWholeModule() const { return !hasPrimaryInputs(); }
 
+  // FIXME: dmu fix uses / remove these when batch mode works
```

Comment:
I commented this on the other PR, but please don't let personal FIXMEs slip into the master branch. You could either remove the note or turn it into a proper, complete-sentence FIXME comment.

Reply:
Sorry! (D'oh!) I have fixed these.